tsdb/ooo_head_read_test.go

```diff
@@ -31,6 +31,12 @@ import (
 	"github.com/prometheus/prometheus/tsdb/wlog"
 )
 
+// Type assertions.
+var (
+	_ chunkenc.Iterable = &mergedOOOChunks{}
+	_ IndexReader       = &HeadAndOOOIndexReader{}
+)
```

Comment:
I don't agree with this change, and I don't see it justified in the PR description.
Such assertions are useful documentation, and tests are stripped out when packages are imported.

Reply:
ack, I'll remove it here and from the OOO  NH PR